app.js

```diff
@@ -13,4 +16,268 @@ navLinks.forEach((link) => {
+  },
+];
+
+cardContent.forEach((item) => {
+  mainContainer.innerHTML += `<div class="content">
+    <img class="item item1" src="${item.images.contentImage}" alt="first project image">
+    <div class="card">
+        <h2 class="item item2">${item.title}</h2>
+        <div>
+            <ul class="row">
+                <li class="item item3">${item.fields[0]}</li>
+                <li><img class="item item4" src=" ${item.images.counter}" alt="a dot image"></li>
+                <li class="item item5">${item.fields[1]}</li>
+                <li><img class="item item6" src="${item.images.counter}" alt="second dot image"></li>
+                <li class="item item7">${item.fields[2]}</li>
+            </ul>
+        </div>
+        <p class="item item8">${item.description}</p>
+        <div class="list">
+            <ul class="skills">
+                <li class="html stack"><button>${item.stack[0]}</button></li>
+                <li class="css stack"><button>${item.stack[1]}</button></li>
+                <li class="javascript stack"><button>${item.stack[2]}</button></li>
+            </ul>
+        </div>
+        <div class="click">
+            <button btn-id="${item.id}" class="btn showBtn" type="button">${item.button}</button>
+        </div>
+    </div>
+</div>`;
+});
+
+const windowPopup = [
+  {
+    id: 'cardOne',
+    title: 'Tonic',
+    fields: ['Canopy', 'Back End Dev', '2015'],
+    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec sit amet leo id risus hendrerit auctor. Praesent at nisi venenatis, fringilla lacus ut, interdum urna.',
+    stack: ['html', 'css', 'javascript'],
+    stackTwo: ['github', 'ruby', 'bootstrap'],
+    images: {
+      contentImage: 'images/Snapshoot-Portfolio.png',
+      counter: 'images/Counter.png',
+      liveImage: 'images/see-live.png',
+      sourceImage: 'images/see-source.png',
+      cancelImage: 'images/xicon.png',
+    },
+    btn1: 'See live',
+    btn2: 'See source',
+    navigation: {
+      live: 'https://github.com/shaaibu7/Portfolio-project',
+      source: 'https://github.com/shaaibu7/Portfolio-project',
+    },
+  },
+
+  {
+    id: 'cardTwo',
+    title: 'Multi-Post Stories',
+    fields: ['Canopy', 'Back End Dev', '2015'],
+    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec sit amet leo id risus hendrerit auctor. Praesent at nisi venenatis, fringilla lacus ut, interdum urna.',
+    stack: ['html', 'css', 'javascript'],
+    stackTwo: ['github', 'ruby', 'bootstrap'],
+    images: {
+      contentImage: 'images/Snapshoot-Portfolio1.png',
+      counter: 'images/Counter.png',
+      liveImage: 'images/see-live.png',
+      sourceImage: 'images/see-source.png',
+      cancelImage: 'images/xicon.png',
+    },
+    btn1: 'See live',
+    btn2: 'See source',
+    navigation: {
+      live: 'https://github.com/shaaibu7/Portfolio-project',
+      source: 'https://github.com/shaaibu7/Portfolio-project',
+    },
```

Comment:
- [ ] Great effort working on your pop-up modal 👏 👏, but I would kindly suggest you have only one `array` of an `object` for creating both your main page dynamically and your pop-up page. with this, you will follow the `DRY (Don't Repeat Yourself)`  method of coding practices. Having only one `array of an object`, you will loop through it over again for the pop-up modal using another value.